### DISCUSSION TITLE
Parques eólicos

### DISCUSSION THREAD

**Discussion by manudona92** [manudona92/MapantES#71] [General]
Veo que los parques eólicos están siendo renderizados como privado. 

No me se la actual legislación, pero quizás sin privado sería mejor (?). De memoria me vienen a la cabeza un par de mapas de orientación debajo de parques eólicos pero no se si realmente es privado o no todo el perímetro. Parece ser que esa capa se coge de OSM. 
https://mapant.es/#x=-97837.96&y=4981073.57&resolution=2.8284&layers=vector
GID=306804517 Cob=20

**Comment by jaluzdi** [comment]
Buenas noches
En el rogaine que organizamos (COBI) en Poza de la Sal, hace un par de años, la parte noroeste del mapa ocupaba la parte del páramo, encima del diapiro de Poza de la Sal, y está llena de molinos eólicos. 
Desconozco también la legislación, pero en ese mapa no se pintó nada en verde oliva (privado). 

<img width="697" alt="image" src="https://github.com/manudona92/MapantES/assets/130509439/a4364aad-1dac-432e-a826-a9cffd82077c">

**Comment by agustincaballero** [comment]
Se pueden quitar, sin problema. Bueno, mejor dicho, no generarlos a partir de ahora. Los que ya están generados el problema es que no hay forma de distinguirlos de otros tipos de centrales eléctricas que entiendo que sí que se quiere generar en verde oliva.

**Reply by agustincaballero** [reply]
En cualquier caso, para lo ya generado no hay gran problema, porque debajo del verde oliva está el terreno normal, con blancos, verdes, etc.
